### ISSUE TITLE
AllHiC on Hifiasm assembly

### ISSUE THREAD

**Issue by sandipmkale** [tangerzhang/ALLHiC#112]
Hello,

Thanks for great tool. I have question regarding development of phased assembly. I used hifiasm to generate haplotype level contain assemblies. As the phases are interchanged in the hifiasm assembly, can I merge the *hap1.p.ctg.fa and *hap2.p.ctg.fa and run the AllHiC to generate phased assemblies?

Thanks and regards

Sandip

**Comment by tangerzhang** [comment]
Yes, It should be totally OK to merge the hap1.p_ctg.fa and hap2.p_ctg.fa and run ALLHiC for haplotype phasing. Alternatively, you can also use p_utg.fa for ALLHiC phasing if you do not have trio data.

**Comment by sandipmkale** [comment]
Thanks alot for reply. The haploid genome size is 2.5 Gb, but the *p_utg.fa file is 4.5 Gb so around 500 Mb data is merged in the *p_utg.fa file. I run the AllHiC but didn't get satisfactory results. I calculated coverage for each units using the pipeline mentioned in the tea genome (Nature Genetics) paper but the maximum depth in *read_depth.txt file is 1. so decided to merge  hap1.p_ctg.fa and hap2.p_ctg.fa.
